hudi-spark/src/main/scala/org/apache/hudi/HoodieWriterUtils.scala

```diff
@@ -20,8 +20,8 @@ package org.apache.hudi
 import org.apache.hudi.DataSourceWriteOptions._
 import org.apache.hudi.common.config.TypedProperties
 
-import scala.collection.JavaConversions._
-import scala.collection.JavaConverters._
+import scala.collection.JavaConversions.mapAsJavaMap
```

Comment:
In general we should have a practice of not adding ".*" or "._" unless more than 3 or 4 entities are imported from same package.